tests/models/esm/test_tokenization_esm.py

```diff
@@ -0,0 +1,123 @@
+# coding=utf-8
+# Copyright 2021 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import os
+import tempfile
+import unittest
+from typing import List
+
+from transformers.models.esm.tokenization_esm import VOCAB_FILES_NAMES, ESMTokenizer
+from transformers.testing_utils import require_tokenizers
+from transformers.tokenization_utils import PreTrainedTokenizer
+from transformers.tokenization_utils_base import PreTrainedTokenizerBase
+
+
+@require_tokenizers
+class ESMTokenizationTest(unittest.TestCase):
+    tokenizer_class = ESMTokenizer
+
+    def setUp(self):
+        super().setUp()
+        self.tmpdirname = tempfile.mkdtemp()
+        vocab_tokens: List[str] = [
```

Comment:
Deactivate the formatter here and put everything in one line.

Reply:
Done!